checks/pinned_dependencies.go

```diff
@@ -598,6 +598,16 @@ func validateGitHubActionWorkflow(pathfn string, content []byte,
 				// Cannot check further, continue.
 				continue
 			}
+
```

Comment:
We could  probably have an exception list separately maintained in a file and use that as a list. This is going to come up for other repositories instead of harcoding them .

Reply:
Given that CIFuzz is kind of unique in a way I don't expect other actions to be on that list so I'd keep it simple for the time being. This can be revisited once another action that can't be pinned properly pops up I think

Reply:
I put CIFuzz and CFLite in an array. That's as far as I would go

Reply:
OK, Sounds good. Thanks